build-aux/couchdb-build-release.sh

```diff
@@ -23,6 +23,7 @@ cd src/
 
 for repo in *; do
   cd $repo
+  grep -rl '{vsn, git}' | xargs sed -i "s/{vsn, git}/{vsn, \"${VERSION}-dev\"}/"
```

Comment:
can we grab the actual value here? For users of couchdb 2.0, 2.1, etc, they won't see or care about these values, but other users, like Cloudant, will make releases that only change some applications, so showing the actual git value will help. Specifically, the output of `git describe --always --tags` within that directory.